runtime/parser/declaration.go

```diff
@@ -127,7 +137,7 @@ func parseDeclaration(p *parser, docString string) (ast.Declaration, error) {
 //
 func parseAccess(p *parser) (ast.Access, error) {
 
-	switch p.current.Value {
+	switch string(p.tokenSource(p.current)) {
```

Comment:
`string(p.tokenSource(p.current))` seems to repeat quite a bit below. Maybe move this to a method?
e.g: `p.currentToken()` or `p.currentTokenString()`

Reply:
@SaveTheRbtz Do you happen to know if extracting this into a function is as optimal as performing the temporary string conversion of the byte slice just for the switch?